sdk/storage/azure-storage-queue/CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 ## 12.2.1 (Unreleased)
 
 ### Features Added
-- Added support for `create_if_not_exists()` for `QueueClient`
```

Comment:
Looks like something happened with your merge/diff as your PR is still trying to remove things already in `main`

Reply:
Yeah... Just did another fetch/rebase/push/merge and the CHANGELOG.md just won't accept it.. will look into this more tomorrow